PR_REVIEW.md

```diff
@@ -5,6 +5,16 @@ If the PR is for a standard bid adapter or a standard analytics adapter, just th
 
 For modules and core platform updates, the initial reviewer should request an additional team member to review as a sanity check. Merge should only happen when the PR has 2 `LGTM` from the core team and a documentation PR if required.
 
+### Running Tests and Verifying Integrations
+
+General gulp commands include separate commands for serving the codebase on a built in webserver, creating code coverage reports and allowing serving integration examples. The `review-start` gulp command combinese those into one command.
+
+- Run `gulp review-start`, adding the host parameter `gulp review-start --host=0.0.0.0` will bind to all IPs on the machine
+- Navigate to `http://host-ip:9999`
+- Navigate to build/coverage/lcov-report/index.html to view coverage
+- Navigate to integrationExamples/gpt/hellow_world.html for basic integration testing
+- The hello_world.html and other exampls can be edited and used as needed to verify functionality
+
```

Comment:
@GLStephen Great work, I really like the idea of a reviewer dedicated page. Now, do we wanna get rid of these lines, since running `gulp review-start` will now do all the magic?

Reply:
I'd like to leave them but add them as the "manual" option. I'll clarify it a bit.

Reply:
@Fawke updated